google-cloud-sdk/src/com/google/cloud/tools/intellij/appengine/java/sdk/CloudSdkValidationResult.java

```diff
@@ -32,7 +32,7 @@
       CloudSdkMessageBundle.message("appengine.cloudsdk.java.components.missing")
           + "\n"
           + CloudSdkMessageBundle.message("appengine.cloudsdk.java.components.howtoinstall"),
-      false),
+      true),
```

Comment:
so will now cause _any_ usage of the Cloud Sdk without the java component to show an error to the user?

Reply:
(I don't completely remember the behavior before; if we were requiring the appengine java component for all custom sdk usages).

Reply:
Looks like this field was never used before!

Reply:
It was never required to have App Engine Java to set the custom SDK path, but if you do select custom SDK without App Engine component in Cloud SDK settings, it will show you an error message (not preventing from setting the path though). So I guess we sort of softly required it.

Reply:
ah thats right. so now theres the potential that users that previously were successfully doing things like flex deployments without the java components installed will be getting an error?

Reply:
oh but this PR is only affecting local run..nm